index.html

```diff
@@ -64,10 +64,15 @@ <h2>More</h2>
                         </a>
                     </li> 
                 </ul>
-                <button class="tweet-button dashed-border-green">
-                    Tweet
-                </button>
-                <div class="profile-box three-columns__expand-two dashed-border-green">
+                <div class="java">
+                    <button class="tweet-button dashed-border-green">
+                        Tweet
+                    </button>
+                    <img src="images/twitter/create-tweet.png" alt="tweet-img">
+                </div>
+                <div>
+                </div>
+                <div class="footer-box profile-box three-columns__expand-two dashed-border-green">
```

Comment:
bit of a hack to hide the button and show an image when changing the resolution

Reply:
Will definitely try this myself !